### DISCUSSION TITLE
[BUG] Question Marks trying to enable icons

### DISCUSSION THREAD

**Discussion by madrigal1** [gokcehan/lf#1158] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30615210/225389226-d03781dd-9dcb-4d69-ac5a-dbd6e2ae6a8a.png)
icons file is the icons.example
set icons true is added in lfrc

**Comment by gokcehan** [comment, marked as answer]
@madrigal1 Since you mark this as a bug, please just keep it as an issue without duplication.

**Reply by madrigal1** [reply]
Issue resolved by fixing locale error:
run 
```
LC_ALL=en_US.UTF-8 lf
```

either use locale-gen to fix your verison of linux's locale or alias the above command to lf
